orga/tests/acceptance/sup-organization-participant-list_test.js

```diff
@@ -51,20 +51,20 @@ module('Acceptance | Sup Organization Participant List', function (hooks) {
     module('filters', function () {
       test('it filters students by group', async function (assert) {
         // given
-        const { getByPlaceholderText, findByRole } = await visit('/etudiants');
+        const screen = await visit('/etudiants');
 
         // when
 
-        const select = await getByPlaceholderText('Rechercher par groupe');
+        const select = await screen.getByPlaceholderText('Rechercher par groupe');
```

Comment:
On peut réussir à avoir cet input avec un byRole également, si on peut éviter d'utiliser getByPlaceholderText (c'est pas le pire mais c'est pas l'idéal)

```suggestion
        const select = await screen.getByRole('textbox', { name: 'Entrer un groupe' });
```